apps/src/templates/projects/ProjectCardGrid.jsx

```diff
@@ -86,14 +86,13 @@ class ProjectCardGrid extends Component {
             <ProjectAppTypeArea
               labKey="featured"
               labName={i18n.featuredProjects()}
-              labViewMoreString={i18n.projectTypeDanceViewMore()}
```

Comment:
What does this change do?

Reply:
The public gallery update included not showing the 'View more of ...' button. So this value is not being used since `hideViewMoreLink` has the value of `true`. I left a comment [here](https://github.com/code-dot-org/code-dot-org/blob/8b8bd34cd3a63df769b309e189eaa7b969ede883/apps/src/templates/projects/ProjectAppTypeArea.jsx#L34-L37).